bitbots_ceiling_cam/config/april_tag_settings.yaml

```diff
@@ -0,0 +1,11 @@
+# AprilTag 3 code parameters
+# Find descriptions in apriltag/include/apriltag.h:struct apriltag_detector
+#                      apriltag/include/apriltag.h:struct apriltag_family
+tag_family:        'tag36h11' # options: tag36h11, tag25h9, tag16h5
```

Comment:
perhaps we should change the default family to tag16h5, to have a better orientation detection